### ISSUE TITLE
Proposal - auto PR creation with updated docs on new release

### ISSUE THREAD

**Issue by stijnmoreels** [arcus-azure/arcus#86]
+ we could/should create some action that does the following on each new release:

Create a new PR on the repo with the following changes:
- Cut `./docs/features/**` & `./docs/index.md` to new folder `./docs/vx.x.x` 
- Copy `./docs/preview/features/**` to `./docs/features/**` (overwrite)
- Copy `./docs/preview/index.md` to `./docs/index.md` (overwrite)
- Update new `./docs/index.md` with permalink and redirection.
- Remove old `./docs/vx.x.x/index.md` permalink and redirection.
- Add update `./docs/preview/index.md` with older version `vx.x.x` link.

In short: current `./docs/features` to new `./docs/vx.x.x/features` folder so the `./docs/preview/features` can take the place of `./docs/features`.

In the created PR, we could then add some additional NuGet or other badges manually if not already done.

Something you can help with or have done before @tomkerkhove ?

_Originally posted by @stijnmoreels in https://github.com/arcus-azure/arcus/issues/75#issuecomment-623294636_

**Comment by stijnmoreels** [comment]
The new frontend allows more easier versioning, which makes this rather obsolete. Let's close this.
